boss-ws/src/test/java/org/genomebridge/boss/http/ObjectResourceAcceptanceTest.java

```diff
@@ -570,4 +570,20 @@ public void testCopy() throws Exception {
         // clean up by deleting 2nd object
         checkStatus(OK,delete(client,objectPath2,obj.ownerId));
     }
+
+    @Test
+    public void testMultiResumable() {
+        Client client = new Client();
+        ClientResponse response = checkStatus( CREATED, createObject("testOject", "tdanford", 1001L));
+        String objectPath = checkHeader( response, "Location");
+        ObjectDesc desc = response.getEntity(ObjectDesc.class);
+        response = post(client, objectPath + "/multi",null);
+        assertThat(response.getStatus()).isEqualTo(OK);
+        ResolveResponse rec = response.getEntity(ResolveResponse.class);
+        assertThat(rec).isNotNull();
+        ObjectStoreConfiguration config = RULE.getConfiguration().getCloudStoreConfiguration();
+        String urlToExpect = config.endpoint + '/' + config.bucket + '/' + desc.objectName;
+        assertThat(rec.objectUrl.toString()).startsWith(urlToExpect);
+    }
```

Comment:
This test doesn't check to see that we can actually use the URI to upload, but doing so would involve implementing Google's protocol in the FCS.  That's not too hard, but let's make that a follow-on story on the Scrum board, and call this done.